ghzdac_cal/GHz_DAC_calibrate.py

```diff
@@ -26,9 +33,9 @@
 # GHz_DAC_calibrate is only a front-end, the actual calibration
 # routines are in ghzdac.calibrate
 
-import ghzdac
-import ghzdac.calibrate as calibrate
-import ghzdac.keys as keys
+import servers.ghzdac as ghzdac
+import servers.ghzdac.calibrate as calibrate
+import servers.ghzdac.keys as keys
```

Comment:
I agree that this repo should be reorganized. However, given the current format (and reasonably common practice), I do indeed expect the repo to be inside a directory (i.e. `src`) which is in the python `sys.path`.

Again, I agree that this ought to be fixed.